Bottom: 08549ca50fe07c7aacdb3615c486470d11497fee
Top:    d04cceded3d081d0fa3109124a104a2592a19754
Author: Ben Greear <greearb@candelatech.com>
Date:   2013-01-29 22:48:28 -0800

nfs: Allow up to 5000 NFSv4 connections (instead of 1024).

Signed-off-by: Ben Greear <greearb@candelatech.com>


---

diff --git a/fs/nfs/callback.c b/fs/nfs/callback.c
index a7f2e6e..5bc7446 100644
--- a/fs/nfs/callback.c
+++ b/fs/nfs/callback.c
@@ -317,7 +317,7 @@ static struct svc_serv *nfs_callback_create_svc(int minorversion)
 	/* As there is only one thread we need to over-ride the
 	 * default maximum of 80 connections
 	 */
-	serv->sv_maxconn = 1024;
+	serv->sv_maxconn = 5000;
 	dprintk("nfs_callback_create_svc: service created\n");
 	return serv;
 }
